addons/website_sale/models/product.py

```diff
@@ -102,9 +102,11 @@ def _get_website_pricelists_domain(self, website_id):
         ''' Check above `_is_available_on_website` for explanation.
         Change in this method should be reflected in `_is_available_on_website`.
```

Comment:
_is_available_on_website should also be updated ;)

Reply:
Updated

Reply:
I keep it in mind, but not an emergency since you can easily change the config to make it workable.

It is like if you have a product in company 1 , but published in website 2 that is on company 2 ;) It will not work. But it is just a wrong configuration.

So yes, it will be better to avoid wrong configuration, but for now we can live with that.

I give my priority to real issue that avoid you to sold, or to be paid ;) that change like here than will probably have unconsidered  side effect.

thx